sdk/nodejs/api/utils.ts

```diff
@@ -19,9 +19,13 @@ function buildArgs(args: any): string {
       str.replace(/"/g, "")
     )
 
+  if (args === undefined || args === null) {
+    return ""
+  }
+
   const formattedArgs = Object.entries(args).reduce(
     (acc: any, [key, value]) => {
-      if (value) {
+      if (value !== undefined && value !== null) {
```

Comment:
It doesn't seem to work. There is a cast from object type to string type with 
https://github.com/dagger/dagger/blob/feb296f5aa689542378d9b7daf849f68415028f1/sdk/nodejs/api/utils.ts#L28-L30

When implementing your alternative, the tests fail. In case value is not underfined nor null, then the cast is safe
```shell
+ '{ container { from (address: "alpine") { withExec }} }'
#5 25.07 - '{ container { from (address: "alpine") { withExec (args: ["echo","hello","world"]) }} }'
#5 25.07       + expected - actual
#5 25.07 
#5 25.07       -{ container { from (address: "alpine") { withExec }} }
#5 25.07       +{ container { from (address: "alpine") { withExec (args: ["echo","hello","world"]) }} }
```

Reply:
@helderco, I set it as resolved, even though behavior/implementation is somewhat a bit surprising. I would have thought, as you, the type to be of `string` type. But it might be related to work on objects that Tom did ?